web/src/components/molecules/Integration/IntegrationConnectModal/index.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from "@emotion/styled";
+import React from "react";
```

Comment:
You shouldn't need to import React if you are using the vite react plugin

Reply:
@KaWaite , removed all occurrences of react import.